core/src/test/java/kafka/test/ClusterConfig.java

```diff
@@ -319,7 +302,10 @@ public Builder setPerBrokerProperties(Map<Integer, Map<String, String>> perBroke
         }
 
         public ClusterConfig build() {
-            return new ClusterConfig(type, brokers, controllers, name, autoStart, securityProtocol, listenerName,
+            if (brokers <= 0 || controllers <= 0 || disksPerBroker <= 0) {
```

Comment:
We should enable to set `brokers` to 0 as it is valid to have a cluster with only quorum nodes.

Reply:
btw, those check should be moved to constructor of `ClusterConfig`